wheelhouse.txt

```diff
@@ -2,3 +2,4 @@ charms.templating.jinja2>=1.0.0,<2.0.0
 
 # pin netaddr to avoid pulling importlib-resources from above lib
 netaddr<=0.7.19
+netifaces
```

Comment:
I don't think so. WIthoutit, I'm getting error in etcd install hook
```
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charmhelpers/contrib/network/ip.py", line 39, in <module>
    import netifaces
ModuleNotFoundError: No module named 'netifaces'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charms/reactive/__init__.py", line 71, in main
    bus.discover()
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charms/reactive/bus.py", line 426, in discover
    _register_handlers_from_file(search_path, filepath)
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charms/reactive/bus.py", line 466, in _register_handlers_from_file
    _load_module(root, filepath)
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charms/reactive/bus.py", line 444, in _load_module
    return importlib.import_module(package + module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/lib/juju/agents/unit-etcd-2/charm/reactive/etcd.py", line 43, in <module>
    from etcd_databag import EtcdDatabag
  File "lib/etcd_databag.py", line 8, in <module>
    from etcd_lib import get_ingress_address
  File "lib/etcd_lib.py", line 1, in <module>
    from charmhelpers.contrib.network.ip import is_address_in_network
  File "/var/lib/juju/agents/unit-etcd-2/.venv/lib/python3.8/site-packages/charmhelpers/contrib/network/ip.py", line 46, in <module>
    import netifaces
ModuleNotFoundError: No module named 'netifaces'
```

Reply:
Ah, so it's used in charmhelpers but not listed as an install dependency for that library (I assume since it's part of `contrib` which won't always be used). Fair enough.